app/src/lib/click.ts

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect } from "react"
+
+export function useOnClickOutside(ref: React.RefObject<any>, handler: CallableFunction) {
```

Comment:
any seems ok here. CallableFunction is new too me, looks like it works - I usually use `(event: MouseEvent | TouchEvent) => void`